pkg/ingestor/collector/collector_test.go

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright 2022 The AFF Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package collector
```

Comment:
this seems to be testing the mock, so should be a different file? collector_mock_test.go maybe

Reply:
removed